### ISSUE TITLE
"Add to Collection" button at top of form

### ISSUE THREAD

**Issue by polm** [itchio/itch.io#382]
When I want to add a game to a collection, I usually just add to my default collection. I also don't personally use comments on games in collections right now. However, because the "Add to Collection" button is at the bottom of the screen, I have to scroll down to see the button to submit the form every time.

Obviously this isn't a serious issue, but it would be nice if there was also a submit button at the top of the form to make simple collection addition faster.

**Comment by fasterthanlime** [comment]
Good point! Additionally, I suppose it'd be cool if just pressing the Enter key would validate the form
